src/main/java/github/scarsz/discordsrv/listeners/DiscordChatListener.java

```diff
@@ -90,7 +102,8 @@ public void onGuildMessageReceived(GuildMessageReceivedEvent event) {
         // enforce required account linking
         if (DiscordSRV.config().getBoolean("DiscordChatChannelRequireLinkedAccount")) {
             if (DiscordSRV.getPlugin().getAccountLinkManager() == null) {
-                event.getAuthor().openPrivateChannel().queue(privateChannel -> privateChannel.sendMessage(LangUtil.Message.FAILED_TO_CHECK_LINKED_ACCOUNT.toString()).queue());
+                if (!event.isWebhookMessage())
+                    event.getAuthor().openPrivateChannel().queue(privateChannel -> privateChannel.sendMessage(LangUtil.Message.FAILED_TO_CHECK_LINKED_ACCOUNT.toString()).queue());
                 DiscordUtil.deleteMessage(event.getMessage());
                 return;
             }
```

Comment:
Not directly related to this PR but this check should be skipped for bots, since they don't need a linked account anyway